baseConfig.json

```diff
@@ -291,6 +290,15 @@
         "require-await": "off",
         "sort-keys": "off",
         "symbol-description": "off",
-        "template-tag-spacing": "off"
+        "template-tag-spacing": "off",
+        "for-direction": "off",
+        "no-buffer-constructor": "off",
+        "padding-line-between-statements": [
+            "error",
+            { "blankLine": "always", "prev": "directive", "next": "*" },
+            { "blankLine": "any", "prev": "directive", "next": "directive" }
```

Comment:
I don’t think so. However this is the recommended equivalent of the deprecated rule `lines-around-directive` with settings `[ "error", "always" ]` which we had enabled before.